packages/fxa-content-server/app/tests/spec/views/sms_send.js

```diff
@@ -310,6 +310,31 @@ describe('views/sms_send', () => {
     });
   });
 
+  describe('_isSignedIn', () => {
+    it('returns `true` if the account is signed', () => {
+      sinon.stub(user, 'isSignedInAccount').callsFake(() => true);
+      broker.set('isVerificationSameBrowser', false);
+
+      assert.isTrue(view._isSignedIn());
+      assert.isTrue(user.isSignedInAccount.calledOnce);
+      assert.isTrue(user.isSignedInAccount.calledWith(account));
+    });
+
+    it('returns `true` if verifying in the same browser', () => {
+      sinon.stub(user, 'isSignedInAccount').callsFake(() => false);
+      broker.set('isVerificationSameBrowser', true);
+
+      assert.isTrue(view._isSignedIn());
+    });
+
+    it('returns `false` otherwise', () => {
+      sinon.stub(user, 'isSignedInAccount').callsFake(() => false);
+      broker.set('isVerificationSameBrowser', false);
+
+      assert.isFalse(view._isSignedIn());
+    });
+  });
```

Comment:
Can we add a test for `showSuccessMessage`? It would be nice to have one in place for when the user should see the banner and when it should be hidden. (probably better as a selenium test but a unit test would be OK)

Reply:
`showSuccessMessage` is a simple boolean on the model, so I've updated this suite to test the various scenarios in which the banner should be displayed. As discussed in Slack, I was getting a little lost trying to write this as a Selenium test :(